packages/gatsby-cli/src/reporter/index.js

```diff
@@ -80,7 +80,7 @@ const prematureEnd = () => {
     status: ActivityStatuses.Failed,
   })
 
-  interuptActivities()
+  interruptActivities()
```

Comment:
You need same change few lines below (Line 88)

Reply:
I thought I had checked them all :(

I hate that github doesn't allow an "expand all" so I can quickly check with the browser.

Thanks for checking! :+1: